xarray/core/options.py

```diff
@@ -258,6 +267,8 @@ class set_options:
     warn_for_unclosed_files : bool, default: False
         Whether or not to issue a warning when unclosed files are
         deallocated. This is mostly useful for debugging.
+    time_resolution : {"s", "ms", "us", "ns"}, default: "s"
+        Time resolution used for CF encoding/decoding.
```

Comment:
If possible, we should prefer local control for setting a default time precision.  This would logical to do with encoding, e.g., `encoding['dtype'] = "datetime64[s]"`, or perhaps via keyword argument added to `xarray.decode_cf()`.

Once we do this, do we need this global option?

Reply:
In the first place I thought the global option would be  a neat solution. But it has also it's drawbacks (like with multiprocessing). I'm working towards your suggestion.

Reply:
I've removed the global option completely. Currently this decodes to the lowest possible resolution. I'm now checking if this can be applied via keyword argument (most visible approach I think).

Reply:
@shoyer I've added a `time_unit` kwarg (any suggestion for better naming appreciated) for `decode_cf` and descending functions.

Reply:
ToDO: Note to self: Is time_resolution still needed at this location?

Reply:
Removed docstring